packages/adapters/src/process-instance-adapter.ts

```diff
@@ -61,7 +61,11 @@ IComponent {
                 ? "../../../python/runner/runner.py"
                 : "../../python/runner/runner.py";
 
-            return ["python3", path.resolve(__dirname, runnerPath)];
+            return [
+                "python3",
+                path.resolve(__dirname, runnerPath),
+                "./python-runner-startup.log",
```

Comment:
It's up for the dev to inspect in case runner fails to start/connect, as it may contain valuable info. Perhaps I could log it's contents in case of runner failure, should I do this as part of this PR?

Reply:
Ok, but we'd need to find a solution for all languages. But yes, let's not extend the scope of this PR.